specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/stable/2024-04-03/desktopvirtualization.json

```diff
@@ -5052,6 +5094,42 @@
         }
       }
     },
+    "RegistrationInfoList": {
+      "type": "object",
+      "title": "RegistrationInfoList",
+      "description": "List of RegistrationInfo definitions.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of RegistrationInfo definitions.",
+          "items": {
+            "$ref": "#/definitions/RegistrationInfoMinimal"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "RegistrationInfoMinimal": {
+      "description": "Represents a Minimal set of properties for RegistrationInfo definition.",
+      "type": "object",
+      "properties": {
+        "expirationTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Expiration time of registration token.",
+          "x-nullable": true
+        },
+        "token": {
+          "type": "string",
+          "description": "The registration token base64 encoded string."
+        }
+      }
```

Comment:
Is this a secret ? Please annotate with x-ms-secret if so

Reply:
done

Reply:
@raosuhas out of curiosity how is this annotation handled? Is this something that ARM processes to enforce, is there enforcement done by the SDKs/powershell, or is this just for documentation and the expectation is that the service handles the secret correctly?

Reply:
Autogeneration tools use this annotation to avoid logging this property inadvertently. 

Some more details are here : 
https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret